Source/WebCore/page/scrolling/mac/ScrollingTreeScrollingNodeDelegateMac.mm

```diff
@@ -89,6 +89,12 @@
         m_scrollerPair->verticalScroller().setEnabled(scrollbarEnabledState.verticalScrollbarIsEnabled);
     }
 
+    if (scrollingStateNode.hasChangedProperty(ScrollingStateNode::Property::ShouldPlaceVerticalScrollbarOnLeft)) {
+        auto shouldPlaceVerticalScrollbarOnLeft = scrollingStateNode.shouldPlaceVerticalScrollbarOnLeft();
+        m_scrollerPair->horizontalScroller().setShouldPlaceVerticalScrollbarOnLeft(shouldPlaceVerticalScrollbarOnLeft);
```

Comment:
I am no area expert. I would just like to comment to say that I find it a little confusing to be setting a "Vertical Scrollbar" property on a *horizontal* scroller :)

Reply:
Haha yeah changed to using UserInterfaceLayoutDirection to make this more clear.